CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.8
 
+* [#56139][]:
+
+  Сollection downcasts in cast patterns are now supported. For example,
```

Comment:
Nit: Should there be `:` instead of `,` after “example”?

Reply:
That makes sense, I changed it!